.gitlab-ci.yml

```diff
@@ -15,8 +19,10 @@ test:
   tags:
     - github
   script:
-    - export QVM_URL='http://qvm:5000'
-    - export QUILC_URL='tcp://quilc:5555'
+    - echo "[Rigetti Forest]" > ~/.forest_config
+    - echo "qvm_address = $QVM_URL" >> ~/.forest_config
+    - echo "quilc_address = $QUILC_URL" >> ~/.forest_config
+    - cat ~/.forest_config
```

Comment:
Do you need this line? Do you need both this block of `*_URL` settings and the preceding `variables` block of `*_URL` settings?

Reply:
Yes, I'd like to have the file dumped into the log just in case.